runtime/etc/Makefile.include

```diff
@@ -103,7 +103,7 @@ x:
 	@echo $(CHPL_MAKE_RUNTIME_LIB)
 
 printincludesanddefines:
-	@echo $(CHPL_MAKE_BASE_CFLAGS) $(RUNTIME_DEFS) $(RUNTIME_INCLS)
+	@echo $(CHPL_MAKE_BASE_CFLAGS) $(patsubst -DNDEBUG,,$(patsubst -DCHPL_OPTIMIZE,,$(patsubst -DCHPL_DEBUG,,$(RUNTIME_DEFS)))) $(RUNTIME_INCLS)
```

Comment:
This `make` target gets used for several purposes, it looks like. I think we'll need to have it continue to print all the same things it currently does, and instead in the compiler, remove the undesired `-D` options from its output.

Reply:
I added a commit to leave it alone in `runtime/etc/Makefile.include` and strip them out in the compiler instead.

Some of the other uses of it seem like they might also want to have them removed because they're using it for user-program level things. I didn't change them here though.